dashboard/src/utils/axiosInstance.js

```diff
@@ -11,15 +11,13 @@ let token = Cookies.get("token");
 const axiosInstance = axios.create({
   baseURL: LOCAL_BASE_URL,
   responseType: "json",
-  headers: {
-    Authorization: `Bearer ${token}`,
-  },
```

Comment:
With this removed, do you still want to be fetching the token at line 9, or should that be removed, too?